csds-client/client/client.go

```diff
@@ -0,0 +1,188 @@
+package client
+
+import (
+	"context"
+	"crypto/x509"
+	"flag"
+	"fmt"
+	csdspb "github.com/envoyproxy/go-control-plane/envoy/service/status/v2"
```

Comment:
Can this package be imported using a consistent alias throughout?

Reply:
Sorry. Since this package is only imported here, what does "a consistent alias" mean?

Reply:
This package is imported 3x in your PR, once as csdspb and twice as envoy_service_status_v2.  A single alias would be helpful.

Reply:
Oh I just found that. I will change all of those to `csdspb_v2` in next PR. Since the size of this PR is already too big to review...So sorry about that.

Reply:
Changed all of those to `csdspb`.